main.tf

```diff
@@ -83,6 +83,31 @@ data "google_organization" "org" {
   organization = "${var.org_id}"
 }
 
+resource "null_resource" "check_preconditions" {
```

Comment:
Since we renamed the preconditions script I think we should rename this to be consistent.